qiita_pet/handlers/util.py

```diff
@@ -89,3 +89,32 @@ def to_int(value):
     except ValueError:
         raise HTTPError(400, "%s cannot be converted to an integer" % value)
     return res
+
+
+@execute_as_transaction
+def get_shared_links(obj):
```

Comment:
This is the one from studies. listing_handlers.py has it removed and moved here, since this function is now used in more than just that file.